docs/sources/operators-guide/mimir-runbooks/_index.md

```diff
@@ -1139,6 +1139,34 @@ The limit protects the system’s stability from potential abuse or mistakes, an
 
 > **Note**: Invalid metrics metadata are skipped during the ingestion, and valid metadata within the same request are ingested.
 
+### err-mimir-distributor-max-ingestion-rate
+
+This critical error occurs when the rate of received samples, exemplars and metadata per second is exceeded in a distributor.
```

Comment:
Where is _critical_ defined? What other qualifiers need a definition? What does it mean if I have a _critical_ error? How do I when I have converted it from a _critical_ error to a _non-critical_ error?

Reply:
(I am happy to resolve this question in a common Grafana Labs glossary entry if a common definition is feasible. We can handle this question in general via a subsequent issue elsewhere.)

Reply:
This is good feedback. I'm the culprit, cause I've introduced it without clarifying it. Let's open an issue and address it in a dedicate PR because it affects all runbooks.

Reply:
Opened an issue: https://github.com/grafana/mimir/issues/2005